src/main/kotlin/de/bund/digitalservice/useid/identification/MockDatasource.kt

```diff
@@ -1,23 +1,45 @@
 package de.bund.digitalservice.useid.identification
 
+import mu.KotlinLogging
 import org.springframework.stereotype.Repository
 import reactor.core.publisher.Mono
 import java.util.UUID
 
 @Repository
 class MockDatasource {
     private val sessions = mutableListOf<IdentificationSession>()
+    private val log = KotlinLogging.logger {}
 
-    fun save(tcTokenUrl: String, refreshAddress: String, requestAttributes: List<String>): Mono<IdentificationSession> {
-        val currentSessionId = UUID.randomUUID()
+    fun save(refreshAddress: String, requestAttributes: List<String>): Mono<IdentificationSession> {
```

Comment:
Rename `save` to `create` since "save" indicates that you could also do updates with it (at least that's how Spring uses "save") but here a new session is always added.